.github/ISSUE_TEMPLATE/flaky-test.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Flaky Test
+about: Report a flakey test (one that doesn't pass consistently)
+
+---
+
+## Flakey Test
```

Comment:
s/Flakey/Flaky/

Reply:
Hey, turns out they're both [valid](https://www.merriam-webster.com/dictionary/flaky) spellings, but I'll drop the e here for consistency. Good find.

Reply:
Good to know. I had no idea flakey was also valid, but it stood out as inconsistent.